src/components/molecules/message/message.scss

```
@@ -0,0 +1,32 @@
+@use "../../variables" as *;
+.message_outer {
+    text-align: center;
+    padding: 52px 20px;
+    @media (max-width: $breakpoint-lg) {
+        padding: 32px 20px;
+    }
+    .message_title {
+        font-family: $fontfamily-mincho;
+        font-style: normal;
+        font-weight: 400;
+        word-wrap: break-word;
+        font-size: $fontsize-xl;
+        line-height: 54px;
```

```suggestion
        line-height: 1.5;
```
`line-height` は可能な限り単位なしの数字で設定するようにしましょう！

> <number> (単位なし)
使用値は、この単位のない [<number>](https://developer.mozilla.org/ja/docs/Web/CSS/number) に要素のフォントサイズを掛けたものになります。計算値は、指定された <number> と同じです。ほとんどの場合、継承時の予期しない結果を避けるために、これが line-height を設定する好ましい方法です。

https://developer.mozilla.org/ja/docs/Web/CSS/line-height